tests/functional/test_metrics.py

```diff
@@ -441,6 +441,8 @@ def test_schema_validation_no_namespace(metric, dimension):
     with pytest.raises(SchemaValidationError, match="Must contain a metric namespace."):
         with single_metric(**metric) as my_metric:
             my_metric.add_dimension(**dimension)
+            # Need to discuss if this behavior change is a breaking change
+            my_metric.provider.Validate_Metrics = True
```

Comment:
I didn't test it yet, but yes, I think this is a breaking change. I'll check it in more detail tomorrow.